main/src/ca/uwaterloo/flix/tools/pkg/ManifestParser.scala

```diff
@@ -489,6 +549,8 @@ object ManifestParser {
     }
   }
 
+  private val validName = "[a-zA-Z0-9.:/_-]+".r
```

Comment:
validName should be ValidName
Add documentation to explain what it actually is
Move to top of file